var/spack/repos/builtin/packages/umpire/package.py

```diff
@@ -23,6 +23,8 @@ class Umpire(CachedCMakePackage, CudaPackage, ROCmPackage):
 
     version('develop', branch='develop', submodules=True)
     version('main', branch='main', submodules=True)
+    version('2022.03.01', tag='v2022.03.01', submodules=True)
```

Comment:
```suggestion
    version('2022.03.1', tag='v2022.03.1', submodules=True)
```

Reply:
This should be changed to what @mcfadden8 suggested to match the Umpire release tag, correct?

Reply:
Yup, I already included that change.

Reply:
I see it now. When I looked earlier, it appeared to not be there.